chore: document Kafka Connector 0.8 properties (#1844)
questdb/questdb.io: docs/third-party-tools/kafka/questdb-kafka.md

The connector supports the following configuration options:

| Name                              | Type      | Example                                                     | Default            | Meaning                                                    |
|-----------------------------------|-----------|-------------------------------------------------------------|--------------------|------------------------------------------------------------|

> | topics                            | `string`  | orders,audit                                                | N/A                | Topics to read from                                        |

Reviewer: No space between `orders,audit`. 

Assume it is intentional, but raising in case it is not.

Reviewer: All good -- was just checkin'! I say ship. :)

### Designated timestamps

The connector supports

> [designated timestamps](https://questdb.io/docs/concept/designated-timestamp/). There are three distinct strategies for designated timestamp handling:

Reviewer: ```suggestion
[designated timestamps](https://questdb.io/docs/concept/designated-timestamp/). 

There are three distinct strategies for designated timestamp handling:
```

1. QuestDB server assigns a timestamp when it receives data from the connector. (Default)
1. The connector extracts the timestamp from the Kafka message payload.
1. The connector extracts timestamps from [Kafka message metadata.](https://cwiki.apache.org/confluence/display/KAFKA/KIP-32+-+Add+timestamps+to+Kafka+message)

Kafka messages carry various metadata, one of which is a timestamp. 
To use the Kafka message metadata timestamp as a QuestDB designated timestamp, 
set `timestamp.kafka.native` to `true`.

If a message payload contains a timestamp field, the connector can utilize it
as a designated timestamp. The field's name should be configured using the

> `timestamp.field.name` option. This field should either be an integer or a timestamp.
> When the field is defined as an integer, the connector will automatically detect
> its units. This is applicable for timestamps after `04/26/1970, 5:46:40 PM`. The units
> can also be configured explicitly using the `timestamp.units` option, which supports
> the following values:

Reviewer: ```suggestion
`timestamp.field.name` option. This field should either be an integer or a timestamp.

When the field is defined as an integer, the connector will automatically detect its units. This is applicable for timestamps after `04/26/1970, 5:46:40 PM`. 

The units can also be configured explicitly using the `timestamp.units` option, which supports the following values:
```

- `nanos`
- `micros`
- `millis`
- `auto` (default)

> Note: These 3 strategies are mutually exclusive. It's illegal to set both `timestamp.kafka.native=true` and `timestamp.field.name`.

Reviewer: ```suggestion
Note: These 3 strategies are mutually exclusive. Cannot set both `timestamp.kafka.native=true` and `timestamp.field.name`.
```

👮

Author: Isnt it missing the subject? "Cannot set .." Who cannot set? Is it clear from the context?

### Textual timestamps parsing

> Kafka messages often contain timestamps in a textual format. The connector
> can parse these and use them as timestamps. The field names should be configured
> using the `timestamp.string.fields` option, and the field should be a string.
> The required format can be set using the `timestamp.string.format` option,
> which adheres to the QuestDB timestamp format, see the [QuestDB timestamp](/docs/reference/function/date-time/#date-and-timestamp-format)
> documentation for more details. 

Reviewer: ```suggestion
Kafka messages often contain timestamps in a textual format. The connector can parse these and use them as timestamps. Configure field names as a string with the `timestamp.string.fields` option. Set the timestamp format with the `timestamp.string.format` option, which adheres to the QuestDB timestamp format. 

See the [QuestDB timestamp](/docs/reference/function/date-time/#date-and-timestamp-format) documentation for more details. 
```

Active voice. 💪
